src/frontend/packages/cloud-foundry/src/shared/components/application-preview/application-preview.component.ts

```diff
@@ -0,0 +1,67 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+import { combineLatest, map } from 'rxjs/operators';
+
+import { getFullEndpointApiUrl } from '../../../../../core/src/features/endpoints/endpoint-helpers';
+import { APP_GUID, CF_GUID } from '../../../../../core/src/shared/entity.tokens';
+import { PreviewableComponent } from '../../../../../core/src/shared/previewable-component';
+import { ApplicationService } from '../../../features/applications/application.service';
+import { getGuids } from '../../../features/applications/application/application-base.component';
+
+@Component({
+  selector: 'app-application-preview-component',
+  templateUrl: './application-preview.component.html',
+  styleUrls: ['./application-preview.component.scss'],
+  // useless, necessary to reuse ApplicationService
+  providers: [
+    ApplicationService,
+    {
+      provide: CF_GUID,
+      useFactory: getGuids('cf'),
```

Comment:
If this and APP_GUID will never be populated they could use `useValue` instead and have no depedancies